game-core/src/main/java/games/strategy/engine/chat/Chat.java

```diff
@@ -79,7 +79,8 @@ public void participantAdded(final ChatParticipant chatParticipant) {
     chatters.put(chatParticipant, "");
     updateConnections();
     chatMessageListeners.forEach(
-        listener -> listener.addStatusMessage(chatParticipant.getPlayerName() + " has joined"));
+        listener ->
+            listener.addStatusMessage(chatParticipant.getPlayerName() + " has joined", true));
```

Comment:
Ah yeah, this reminds me, `addStatusMessage`, is NOT EVEN A STATUS!! There is a player 'status' in chat, it's the text that goes in parentheses next to a players name. I forget often that this is actually a `addChatEvent`, and is not at all a player status...

I think we might just need to fix up the API, the boolean parameter is pretty message. EG:
```
listener.joinEvent(PlayerName)
listener.slapEvent(PlayerName)
```

Reply:
Yeah ikr, I confused it too when I was testing stuff.